src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -89,9 +89,9 @@ const useStyles = makeStyles((theme) => ({
     position: 'relative',
     bottom: theme.spacing(20),
     zIndex: 300,
-    margin: '0 auto',
     [theme.breakpoints.between('xs', 'sm')]: {
-      left: '55%',
+      right: theme.spacing(4),
```

Comment:
I think we missed a rebase commit here where you should have accepted remote 😄  https://github.com/Seneca-CDOT/telescope/commit/d2b2e08ca8bd86ea58c1c7d195fb7b686e1bfa5d, this would undo the commit that was previously done.